src/main/webapp/app/exercises/shared/result/result.component.ts

```diff
@@ -194,6 +237,7 @@ export class ResultComponent implements OnInit, OnChanges {
                     return ResultTemplateStatus.SUBMITTED_WAITING_FOR_GRADING;
                 } else {
                     // the due date is over, further submissions are no longer possible, no result after assessment due date
+                    // TODO why is this distinct from the case above? The submission can still be graded and often is.
```

Comment:
In this specific case or in general?

Reply:
This specific case, but I am not sure what you mean here. I can create an issue for that as well.

Reply:
Done, see #4004.